src/Simple/JSON.purs

```diff
@@ -182,7 +182,7 @@ instance writeForeignForeign :: WriteForeign Foreign where
   writeImpl = id
 
 instance writeForeignUnit :: WriteForeign Unit where
```

Comment:
This instance should not exist at all, there is not valid value for Unit. This will also break in any non-JS backend like purerl.

Reply:
Then we'd need to use -generics-rep, right?